src/workers/newNotificationV2Mail.ts

```diff
@@ -663,6 +666,9 @@ const notificationToTemplateData: Record<NotificationType, TemplateDataFunc> = {
       },
     };
   },
+  squad_public_submitted: null,
+  squad_public_rejected: null,
+  squad_public_approved: null,
```

Comment:
Along with the IDs, the required parameters will be provided by the product team.